ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.txt

```diff
@@ -179,12 +188,241 @@ hideall "--sync--"
+3104.1 "Heavy Impact 3" sync / 1[56]:[^:]*:Ser Guerrique:63D8:/
+3106.0 "Heavy Impact 4" sync / 1[56]:[^:]*:Ser Guerrique:63D9:/
+3107.9 "Heavy Impact 5" sync / 1[56]:[^:]*:Ser Guerrique:63DA:/
+3114.4 "The Dragon's Gaze" sync / 1[56]:[^:]*:King Thordan:63D0:/
+3115.5 "Faith Unmoving" #sync / 1[56]:[^:]*:Ser Grinnaux:62DC:/
+3115.5 "The Dragon's Gaze" #sync / 1[56]:[^:]*:King Thordan:63D1:/
+3115.5 "The Dragon's Glory" #sync / 1[56]:[^:]*:King Thordan:63D2:/
+3117.4 "Heavensflame" sync / 1[56]:[^:]*:Ser Charibert:62DE:/
+3140.3 "Ancient Quaga" sync / 1[56]:[^:]*:King Thordan:63C6:/
+3150.5 "Heavenly Heel" sync / 1[56]:[^:]*:King Thordan:63C7:/
+3153.6 "Ascalon's Might 1" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3155.2 "Ascalon's Might 2" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+3156.8 "Ascalon's Might 3" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+# Enrage?
+
+
+# Phase 6: Nidhogg v2
+3500.0 "--sync--" sync / 1[56]:[^:]*:King Thordan:6667:/ window 500,0
+3535.0 "Great Wyrmsbreath" sync / 1[56]:[^:]*:Hraesvelgr:6D35:/
+3535.0 "Dread Wyrmsbreath" #sync / 1[56]:[^:]*:Nidhogg:6D33:/
+3535.7 "Swirling Blizzard" #sync / 1[56]:[^:]*:Hraesvelgr:6D38:/
+3535.8 "Ice Breath" #sync / 1[56]:[^:]*:Hraesvelgr:6D37:/
+3535.8 "Holy Breath" #sync / 1[56]:[^:]*:Hraesvelgr:6D3C:/
+3535.8 "Flame Breath" #sync / 1[56]:[^:]*:Nidhogg:6D36:/
+3535.8 "Staggering Breath" #sync / 1[56]:[^:]*:Nidhogg:6D3D:/
+3543.0 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D30:/
+3554.3 "Akh Afah" sync / 1[56]:[^:]*:Hraesvelgr:6D42:/
+3567.4 "Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D24:/
+3568.5 "Cauterize" sync / 1[56]:[^:]*:Nidhogg:6D3E:/
+3567.8 "Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D27:/
+3568.6 "Hallowed Plume" #sync / 1[56]:[^:]*:Hraesvelgr:6D29:/
+3568.6 "Hallowed Wings" #sync / 1[56]:[^:]*:Hraesvelgr:6D28:/
+3577.0 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3582.4 "Wroth Flames" sync / 1[56]:[^:]*:Nidhogg:6D45:/
+3593.2 "Akh Morn 1" sync / 1[56]:[^:]*:Nidhogg:6D46:/
+3593.3 "Cauterize" #sync / 1[56]:[^:]*:Hraesvelgr:6D3F:/
+3594.9 "Akh Morn 2" sync / 1[56]:[^:]*:Nidhogg:6D47:/
+3596.5 "Akh Morn 3" sync / 1[56]:[^:]*:Nidhogg:6D47:/
+3598.1 "Akh Morn 4" sync / 1[56]:[^:]*:Nidhogg:6D47:/
+3605.2 "Hot Wing" sync / 1[56]:[^:]*:Nidhogg:6D2B:/
+3606.3 "Hot Wing" sync / 1[56]:[^:]*:Nidhogg:6D2C:/
+3607.2 "Spreading Flames" #sync / 1[56]:[^:]*:Nidhogg:742B:/
+3607.2 "Entangled Flames" #sync / 1[56]:[^:]*:Nidhogg:742C:/
+3607.2 "Spreading Flames" #sync / 1[56]:[^:]*:Nidhogg:742B:/
+3607.2 "Entangled Flames" #sync / 1[56]:[^:]*:Nidhogg:742C:/
+3611.1 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3623.4 "Akh Afah" sync / 1[56]:[^:]*:Hraesvelgr:6D42:/
+3635.4 "Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D26:/
+3635.4 "Hot Tail" #sync / 1[56]:[^:]*:Nidhogg:6D2D:/
+3635.5 "Hallowed Plume" #sync / 1[56]:[^:]*:Hraesvelgr:6D29:/
+3635.5 "Hallowed Wings" #sync / 1[56]:[^:]*:Hraesvelgr:6D25:/
+3636.4 "Hot Tail" #sync / 1[56]:[^:]*:Nidhogg:6D2E:/
```

Comment:
It looks like this spell appears in the timeline even if no one gets hit.

Reply:
From a timeline perspective, I really only am differentiating between self-casted and (potential) damage on players.  I think it's weird to have two listings in the log for the same ability (hot tail doesn't happen twice, and I think it's confusing), so my general rule is to leave the self-targeted one (often because that's when mit needs to be applied, although that doesn't matter here) and then skip the damage one.  The more important thing is that I don't think there should be two.